src/Core/Infrastructure/RealTime/Repository/Downtime/DbReadDowntimeRepository.php

```diff
@@ -46,6 +46,14 @@ public function findOnGoingDowntimesByHostId(int $hostId): array
         return $this->findOnGoingDowntimes($hostId, 0);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function findOnGoingDowntimesByHostIdAndServiceId(int $hostId, int $serviceId): array
+    {
+        return [];
+    }
+
```

Comment:
you can mock this return in your test, so it could be not necessary to implement this method in this PR.